### PR TITLE
Searchbar

### PR DESCRIPTION
**Pull request by pascalweber85**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,3 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This is Leipzig. Have fun to explore the "Best Locations" in the Town and add it to your Favorites! Also you can see more "Details" by clicking the pictures.
```
Comment (roscha80) [inline README.md:3]: Maybe some screenshots of the app would looks cool there :)

**Review by roscha80 — APPROVED** [review]
On Click on Entdecken and Favoriten is still a console.log which displays a function in the console. In the browser there is still display React App at the tab ;)

File: src/pages/DetailsPage.js [hunk 2]
```diff
@@ -3,23 +3,26 @@ import styled from 'styled-components/macro'
 import Button from './../components/FavButton'
 import { useHistory } from 'react-router-dom'
 import Icon from './../Images/back.png'
+import { useParams } from 'react-router-dom/cjs/react-router-dom.min'
+// import Slider from '../components/Slider'
```
Comment (hugh-burgess) [inline src/pages/DetailsPage.js:7]: You can delete these commented code parts here

File: src/pages/DetailsPage.js [hunk 3]
```diff
@@ -3,23 +3,26 @@ import styled from 'styled-components/macro'
...
+  const {
+    title,
+    main_image_path,
+    öffnungszeiten,
+    beschreibung,
+  } = locations.find(location => location.id === id)
 
   return (
     <Wrapper>
+      {/* <Slider images={imageArray} /> */}
```
Comment (hugh-burgess) [inline src/pages/DetailsPage.js:25]: This too

File: src/pages/StartPage.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ export default function StartPage() {
       <h1>
         <span></span>
       </h1>
-
+      {/* <Slider images={imageArray} /> */}
```
Comment (hugh-burgess) [inline src/pages/StartPage.js:19]: This can go I think

**Review by hugh-burgess — COMMENTED** [review]
Well done mate you’re almost done! Just some little commented code I found. LGTM!

**Pull request merged**